src/config.js

```diff
@@ -54,7 +59,37 @@ function loadConfig() {
 
   conf.validate();
 
-  return conf.getProperties();
+  const resolvedConfig = conf.getProperties();
+
+  validateConfigValues(resolvedConfig);
+
+  return resolvedConfig;
+}
+
+// This checks if the configured values are valid, and throws otherwise.
+function validateConfigValues(config: Config) {
+  const { googleAuthenticationFilePath } = config;
+
+  if (
+    googleAuthenticationFilePath &&
+    googleAuthenticationFilePath !== 'MOCKED'
+  ) {
+    // If some other value is set, we check it ahead of time so that the lib
+    // doesn't output an error later.
+    // Note that an empty string is a valid value: in that case the lib will try
+    // to find the authentication information from other means.
+    try {
+      fs.accessSync(googleAuthenticationFilePath, fs.constants.R_OK);
```

Comment:
I use the synchronous version of the function because this is running at start time.